Library/Homebrew/cask/lib/hbc/utils/quarantine.swift

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/swift
+
+import Foundation
+import CoreServices
+
+let QUARANTINE_SUCCESS: Int = 0
+let QUARANTINE_FAILURE: Int = 1
+
+if (CommandLine.arguments.count < 3) {
+  print("Insufficient parameters: this script needs file_path and data_url")
+  exit(Int32(QUARANTINE_FAILURE))
+}
+
+let filepath = CommandLine.arguments[1]
+let agent: String = "Homebrew-Cask"
+let bundle: String = "sh.brew.cask"
+let dataUrl: String = CommandLine.arguments[2]
+
+let quarantineProperties: NSMutableDictionary = NSMutableDictionary()
```

Comment:
Can this be a normal Swift dictionary which is casted afterwards?

Reply:
Yes, will do.